rust/python_bindings/rust_bindings.pyi

```diff
@@ -12,10 +11,15 @@ from chromadb.api.types import (
     URIs,
     Include,
 )
-from chromadb.types import Database, Tenant, Collection as CollectionModel
+from chromadb.types import Tenant, Collection as CollectionModel
 from chromadb.config import DEFAULT_DATABASE, DEFAULT_TENANT
 from enum import Enum
 
+class DatabaseFromBindings:
```

Comment:
review: should this just be called `Database`? didn't want to collide with the existing `TypedDict` def